src/content/docs/en/pages/products/edge-firewall/rules-engine.mdx

```diff
@@ -106,7 +107,8 @@ These are all the available behaviors:
 | Set Rate Limit                | It defines an access rate limit that, if exceeded, will result in an HTTP *429 Too Many Requests* response. To configure the *Rate Limit*, you must inform:<br />`Type` is the field to select the type of request for the Rate Limit, choosing `Req/s` (request per second) or `Req/Min` (request per minute). <br />`Average Rate Limit`, which is the limit rate itself by second. <br />`Client IP address`, if you want the access rate to be counted by the client's IP address or `Global`, if you want total access rate counted. <br />`Maximum burst size`, which indicates the maximum burst size of HTTP requests sent simultaneously, which will be queued and dispatched gradually, respecting the limit rate.<br />The configured value will be the Rate Limit for each Azion Edge Node, implemented using the Leaky Bucket algorithm. It's recommended to use `Maximum burst size` at a maximum of 10 times of the value configured in `Average Rate Limit`, which would result in penalizing the last request for a burst with up to a 10 second delay. | -                        |
 | Set WAF Rule Set              | It associates the WAF Rule Set to be used in the request. WAF policies must be previously configured in the *Edge Libraries > WAF Rules*. | Web Application Firewall |
 | Run a Function                | It runs a function specified as a parameter. The function must have been previously instantiated and parameterized in the Functions tab in order to be used. | Edge Functions           |
+| Set Custom Response           | Allow a customized response when the request matches the criteria. You can customize the `status_code`, by changing it from 200 to 499; `Content-Type` header; and `body` of your request, with a maximum of 500 characters. | - | 
```

Comment:
```suggestion
| Set Custom Response           | It allows a customized response when the request matches the criteria. You can customize the `status_code` by changing it from 200 to 499, `Content-Type` header, and `body` of your request with a maximum of 500 characters. | - | 
```

Reply:
I suggest to change from `status_code` to `Status Code` and `body` to `Content Body`, like the field names from Edge Firewall Rules Engine
![2023-09-28_15-10](https://github.com/aziontech/docs/assets/54641406/c5db68e7-8384-496d-8484-6d0ebce85394)

Reply:
Great suggestion, Furno. Thanks!